modules/xfem/src/efa/EFAElement3D.C

```diff
@@ -1957,6 +2088,27 @@ EFAElement3D::addFaceEdgeCut(unsigned int face_id,
       face_neighbor->addFaceEdgeCut(
           neigh_face_id, neigh_edge_id, neigh_pos, local_embedded, EmbeddedNodes, false, true);
     }
+
+    for (unsigned int en_iter = 0; en_iter < numEdgeNeighbors(face_id, edge_id); ++en_iter)
```

Comment:
If you made that change I suggested about changing the type of _face_edge_neighbors to have a set, you would probably want to replace `getEdgeNeighbor()` with a function that gets the set, and then iterate over it here.

Reply:
We will make the cleanup work on both _face_edge_neighbors and _face_neighbors in the future.